src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1720,7 +1720,7 @@
     <value>An assembly has to opt into preview features before using them.</value>
```

Comment:
This PR is not ready for review yet. I need to clean it up a little bit,but I wanted to see how CI performs with the CultureInfo changes first.